great_expectations/data_context/data_context/ephemeral_data_context.py

```diff
@@ -74,21 +70,3 @@ def _init_datasource_store(self) -> DatasourceStore:
         )
 
         return datasource_store
-
-    @public_api
-    def convert_to_file_context(self) -> FileDataContext:
```

Comment:
Removing this since it gets mad about `self.stores` and it is slated for deletion